src/Lesson5/FirstProgram.java

```diff
@@ -0,0 +1,29 @@
+package Lesson5;
+
+import java.util.Scanner;
+
+public class FirstProgram {
+
+    public static void main(String[] args) {
+
+        //The first program
+
+        Scanner scr = new Scanner(System.in);
+
+        System.out.println("Enter the first number: ");
+        int first = scr.nextInt();
+
+        System.out.println("Enter the second number: ");
+        int second = scr.nextInt();
+
+        boolean firstOption = first > second;
```

Comment:
what does it mean 'firstOption' ? can be something like 'isSecondLessThanFirst'
and what if first equals second?

Reply:
"firstOption" this is a convention.  Initially I wanted to make 3 programs in one class and break it into options inside the class instead of the 3 program.

I wanted to make an option if the numbers are equal.  But in the assignment it was specified specifically to find a larger number or a smaller number.  Redo with the addition of the option if the numbers are equal ?

Reply:
Add changes. Check please https://github.com/aqaman0011/myHomeworks/pull/5/commits/3d5e572264ebc88ac36a9cad8d9f9322c33933a2